project-templates/import-records/Readme

```diff
@@ -0,0 +1,122 @@
+Requirements
+
+  node >= 4 https://nodejs.org/
+  make      https://www.gnu.org/software/make/
+  jq        https://stedolan.github.io/jq/
+
+Make does not support filenames with special characters unfortunately.  Remove
+special characters in your filenames.
+
+Make is not included on OSX by default, you need to install XCode or use a
+package manager like [Brew](https://brew.sh/) or
+[Nix](https://nixos.org/nix/).
+
+Import Steps:
+
+  1) Gather source data, export from a database into linked spreadsheets.  By
+     linked I mean related data in two spreadsheets can be joined with a shared
+     column.
+
+  2) Format and convert
+
+    PLACES_LEVEL_3=households.csv \
+    PEOPLE=people.csv \
+    RECORDS=seguimiento.csv \
+    make
+
+  3) Dry run of import with subset (one row) of data:
+
+    export CR_PROD=https://admin:secret@myproject.app.medicmobile.org
+
+    COUCH_URL=$CR_PROD \
+    PLACES_LEVEL_3_FIELDMAP=households-fieldmap.csv \
+    PLACES_LEVEL_3_PEOPLE_FIELDMAP=households-people-fieldmap.csv \
+    RECORDS_FIELDMAP=seguimiento-fieldmap.csv \
+    make import
```

Comment:
I don't think this example is actually a dry run? Looks like a real run with all data